.circleci/config.yml

```diff
@@ -279,6 +285,9 @@ commands:
             echo 'export AWS_CRYPTO_CONTEXT_STAGE=$AWS_CRYPTO_CONTEXT_STAGE_STAGING' >> $BASH_ENV
             echo 'export AWS_CRYPTO_CONTEXT_PURPOSE=$AWS_CRYPTO_CONTEXT_PURPOSE' >> $BASH_ENV
             echo 'export AWS_CRYPTO_CONTEXT_ORIGIN=$AWS_CRYPTO_CONTEXT_ORIGIN' >> $BASH_ENV
+            echo 'export USE_BASIC_AUTH=$USE_BASIC_AUTH_STAGING' >> $BASH_ENV
+            echo 'export BASIC_AUTH_USERNAME=$BASIC_AUTH_USERNAME_STAGING' >> $BASH_ENV
+            echo 'export BASIC_AUTH_PASSWORD=$BASIC_AUTH_PASSWORD_STAGING' >> $BASH_ENV
```

Comment:
For this and also the dev section, I don't think we need to have separate usernames and passwords for dev and staging. So there's no need to have BASIC_AUTH_PASSWORD_STAGING and BASIC_AUTH_PASSWORD_DEV as two separate environment variables